index.js

```diff
@@ -4,8 +4,15 @@ const express = require('express'),
       session = require('express-session'),
       displayRoutes = require('express-routemap'),
       bcrypt = require('bcrypt'),
+      crypto = require('crypto'),
```

Comment:
crypto is a native node.js package that provides cryptographic utilities. Bcrypt is a crypotographic algorithm that is not included in the node.js by default.